alerts/mssql-server/connections-near-limit.json

```diff
@@ -0,0 +1,38 @@
+{
+    "displayName": "MSSQL - Connections Near Limit",
+    "documentation": {
+        "content": "If the amount of connections is within 10% of the maximum connections, default being 32,767 (threshold is defaulted to 29490), then that could be cause to look for ways to increase performance and load management.",
+        "mimeType": "text/markdown"
+    },
+    "userLabels": {},
```

Comment:
```suggestion
```
Suggest removing the empty object here for simplicity.  This comment applies for other empty arrays in other files.

Reply:
Sorry for the back-and-forth, could we actually keep this empty array? User labels are a common addition made by our users and we want to make it easy for them to discover that it's possible to add tags to their alert policies.